### ISSUE TITLE
Suggestion: Make "Vote Submitted" Pop Up, or some other clear indication that the vote went through

### ISSUE THREAD

**Issue by laurenluz** [wslyvh/tokenlog#32]
It took me a minute to realize that my vote went through for the MVV because the indicator was so unremarkable. Would be amazing if, in active window, it could say "Vote Submitted!" in red... or there could be another popup or indicator.

The change in the text below is not very noticeable:

![image](https://user-images.githubusercontent.com/75490971/116286458-7bb8e500-a74c-11eb-8e40-c134265dd49d.png)

**Comment by wslyvh** [comment]
That makes sense. Will look into this.

**Comment by wslyvh** [comment]
Added 2 status indicators when you're casting your vote.
I noticed casting the vote itself takes quite some time, so starting with a loading indicator first.

![image](https://user-images.githubusercontent.com/25974464/116782701-00786b80-aa8b-11eb-9cb7-154658778ee3.png)

Once successfully completed, you'll get a notification that it has been submitted.
![image](https://user-images.githubusercontent.com/25974464/116782706-079f7980-aa8b-11eb-9628-971a04a37a9c.png)
